admin_guide/topics/pod-priority-about.adoc

```diff
@@ -11,6 +11,11 @@ Module included in the following assemblies:
 
 When the Pod Priority and Preemption feature is enabled, the scheduler orders pending pods by their priority, and a pending pod is placed ahead of other pending pods with lower priority in the scheduling queue. As a result, the higher priority pod might be scheduled sooner than pods with lower priority if its scheduling requirements are met. If a pod cannot be scheduled, scheduler continues to schedule other lower priority pods.
 
+[NOTE]
+====
+Priority pods can only run in reserved namespaces, such as `kube-system`.
```

Comment:
We allow critical pods to be created in `openshift-*` namespaces as well.

https://github.com/openshift/origin/blob/release-3.11/vendor/k8s.io/kubernetes/plugin/pkg/admission/priority/admission.go#L153